pitivi/utils/pipeline.py

```diff
@@ -613,7 +613,7 @@ def simple_seek(self, position):
 
     def _busMessageCb(self, bus, message):
         if message.type == Gst.MessageType.ASYNC_DONE:
-            self.commiting = False
```

Comment:
Oops! Many thanks!

Reply:
Hm, actually it seems to be a field which has never been used. I removed it in a separate commit.